src/components/modals/login-modal/LoginContent.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Form, Input } from 'antd';
+import React from 'react';
+import './login-content.less';
+
+interface Props {
+  switchToForgotPasswordPage: () => void;
+}
+
+const LoginContent: (props: Props) => JSX.Element = (props: Props) => {
+  const forgotPasswordLink = (
+    <p onClick={props.switchToForgotPasswordPage} className="page-link">
+      here
+    </p>
+  );
+
+  const signUpLink = <a className="page-link">here</a>; // eslint-disable-line no-eval
+
+  return (
+    <p>
```

Comment:
We should avoid to nest elements in <p> tags. In general we should never use <p> tags and use AntD typography instead!